styleguide/components/31-molecules/building-blocks/quote/_quote.scss

```diff
@@ -16,13 +16,11 @@
   blockquote {
     position: relative;
     flex: 1;
-
-    @include desktop {
-      padding-left: 9rem;
-    }
+    max-width: 40rem;
```

Comment:
I've added this in base somewhere for all text elements.
Let's keep it together.